web/src/util/roundedPercents.test.js

```diff
@@ -0,0 +1,14 @@
+import roundedPercents from './roundedPercents';
+
+describe('rounding percentages util', () => {
+  test('spreads a number across a set of percentages', () => {
+    expect(roundedPercents(224, [0.15, 0.19, 0.3, 0.36])).toEqual([
+      34,
+      42,
+      67,
+      81
+    ]);
+
+    expect(roundedPercents(100, [1 / 3, 1 / 3, 1 / 3])).toEqual([33, 33, 34]);
```

Comment:
And these because 1/3 is funky.  This test actually found a bug where I was taking the floor of a number instead of rounding it, so I was actually summing to 99.